apis/python/tests/test_factory.py

```diff
@@ -182,11 +136,18 @@ def test_factory_unknown_files():
 
 
 def _setmetadata(open_tdb_object, metadata_typename, encoding_version):
-    """set only those values which are not None"""
-    changes = {}
+    """force modify the metadata values"""
+    set_metadata = open_tdb_object._handle._handle.set_metadata
```

Comment:
i really like this b/c we get access to the `force` flag at the `object._handle._handle` level but it does not work its way up to the the API level (which is good)